specs/electra/fork.md

```diff
@@ -74,10 +74,12 @@ def upgrade_to_electra(pre: deneb.BeaconState) -> BeaconState:
     epoch = deneb.get_current_epoch(pre)
     latest_execution_payload_header = pre.latest_execution_payload_header
 
-    exit_epochs = [v.exit_epoch for v in pre.validators if v.exit_epoch != FAR_FUTURE_EPOCH]
-    if not exit_epochs:
-        exit_epochs = [get_current_epoch(pre)]
-    earliest_exit_epoch = max(exit_epochs) + 1
+    earliest_exit_epoch = compute_activation_exit_epoch(get_current_epoch(pre))
```

Comment:
maybe:
```python
earliest_exit_epoch = max(
    [v.exit_epoch for v in pre.validators if v.exit_epoch != FAR_FUTURE_EPOCH] +
    [compute_activation_exit_epoch(get_current_epoch(pre))]
)
```
I don’t think the `+= Epoch(1)` is needed since we have `compute_activation_exit_epoch`

Reply:
Hmm I think that code is a bit too complex/pythonic. Clients would have trouble following this exactly.

I believe the `+= Epoch(1)` is necessary if we want similar functionality as before. `compute_activation_exit_epoch` only sets the lower bound.

Reply:
Yes, I agree. It should be `+1` to ensure that the exit mechanism is reset in case when the `earliest_exit_epoch` is far in the future. I am happy to leave the code as is and appreciating the argument on not making the spec too pythonic.